exe/server/main.go

```diff
@@ -63,15 +63,20 @@ func main() {
 		if err != nil {
 			log.Fatal(err)
 		}
-	}
 
-	repoPath := config.GetString("repopath")
-	if repoPath == "${HOME}/.texfc" {
-		home, err := os.UserHomeDir()
+		repoPath := config.GetString("repopath")
+		if repoPath == "${HOME}/.texfc" {
+			home, err := os.UserHomeDir()
+			if err != nil {
+				log.Fatalf("error when setting default repo path to home dir: %s", err)
+			}
+			repoPath = strings.Replace(repoPath, "${HOME}", home, -1)
+		}
+	} else {
+		repoPath, err = ioutil.TempDir("", ".texfc-*")
 		if err != nil {
-			log.Fatalf("error when setting default repo path to home dir: %s", err)
+			log.Fatal(err)
 		}
-		repoPath = strings.Replace(repoPath, "${HOME}", home, -1)
 	}
```

Comment:
Yeah, I know what are you referring to.  But this change avoids that error.
A bit more detail on the situation:
When the embeded network is run, the Lotus node uses `os.TempDir()` as its base path for the repo. Since this solution in `--embedded` mode also uses `os.TempDir()`, that constrain of using the same  _IPFS_PATH_ holds correctly.